usaspending_api/database_scripts/matview_generator/mv_directpayment_award_search.json

```diff
@@ -285,7 +192,6 @@
         ") AND recipient_name IS NOT NULL",
         "AND recipient_level != 'P'",
       "GROUP BY recipient_hash, recipient_unique_id",
-      "LIMIT 1",
```

Comment:
I would flip the question around and ask "If it is limiting the subquery to a single row when there are multiple, how is the sql deterministically picking the _best_ one without any ordering?"

But to your point, based on the predicate and grouping I didn't understand how there could be multiple rows returned. It would be easy to re-add if there are problems

Reply:
Good flip. I didn't look closely at the query but thought it would have had a `DISTINCT ON` where it was targeting the top row of ordered results, but it doesn't appear to be the case now that I look. So i'm good with the change here.